docs/misc/state_sync_from_s3.md

```diff
@@ -0,0 +1,59 @@
+# Experimental: Sync state from External Storage
```

Comment:
At the risk of sounding overly negative... could we update this to mention that this is meant to be temporary? As it is, this doc makes it sound like syncing from S3 is the new thing we're going for, when hopefully it's just temporary, and we'll be able to get rid of it when we implement a better version of peer-to-peer state sync. My worry with this is that if we get too comfortable with this nice and fast and reliable new S3 sync mechanism, we'll forget that there's something in here that needs fixing ASAP. It's not like a problem that causes your node to crash all the time, which very obviously needs fixing. Here we'll have something that almost always works great, but which should be treated as needing somewhat timely fixing in my opinion

Reply:
Agreed.
Reworded to make it clear that this solution will be replaced with a decentralized solution.